demo/tool/README.md

```diff
@@ -0,0 +1,215 @@
+# Check for updated demo files in GitHub organizations and repositories
+
+## Update tool
+
+### </a>Prepare
```

Comment:
I made this to be able to link to the caption from farther below. I looked it up somewhere and don't know about another way. I tried it by opening the file using the "View" button and it works.